### PR TITLE
Update antrea YAML with latest udp offload stuff

### PR DESCRIPTION
**Pull request by jayunit100**
## What this PR does / why we need it 

Adding UDP offload 

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
support UDP offload configmaps
```

### CODE REVIEW

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 1]
```diff
@@ -3872,20 +3908,20 @@ data:
...
-    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+    # Valid time units are "ns", "us" (or "Âµs"), "ms", "s", "m", "h".
     #flowPollInterval: "5s"
 
     # Provide the active flow export timeout, which is the timeout after which a flow
     # record is sent to the collector for active flows. Thus, for flows with a continuous
     # stream of packets, a flow record will be exported to the collector once the elapsed
     # time since the last export event is equal to the value of this timeout.
-    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+    # Valid time units are "ns", "us" (or "Âµs"), "ms", "s", "m", "h".
```
Comment (jayunit100) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:3918]:
```suggestion
    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 2]
```diff
@@ -3982,12 +4020,37 @@ data:
...
+
+    # Mask size for IPv4 Node CIDR in IPv4 or dual-stack cluster. Value ignored when enableNodeIPAM is false
+    # or when IPv4 Pod CIDR is not configured. Valid range is 16 to 30.
+    #  nodeCIDRMaskSizeIPv4: 24
+
+    # Mask size for IPv6 Node CIDR in IPv6 or dual-stack cluster. Value ignored when enableNodeIPAM is false
+    # or when IPv6 Pod CIDR is not configured. Valid range is 64 to 126.
+    #  nodeCIDRMaskSizeIPv6: 64
+
+    enterpriseAntrea: true
```
Comment (jayunit100) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:4047]:
```suggestion
    enterpriseAntrea: false
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 3]
```diff
@@ -3901,8 +3937,7 @@ data:
     # https://golang.org/pkg/crypto/tls/#pkg-constants
     # Note that TLS1.3 Cipher Suites cannot be added to the list. But the apiserver will always
     # prefer TLS1.3 Cipher Suites whenever possible.
-    #tlsCipherSuites:
-
+    tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
```
Comment (knabben) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:3940]:
```suggestion
    #tlsCipherSuites: 
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 4]
```diff
@@ -3964,7 +4002,7 @@ data:
     # https://golang.org/pkg/crypto/tls/#pkg-constants
     # Note that TLS1.3 Cipher Suites cannot be added to the list. But the apiserver will always
     # prefer TLS1.3 Cipher Suites whenever possible.
-    #tlsCipherSuites:
+    tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
```
Comment (knabben) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:4005]:
```suggestion
    #tlsCipherSuites: 
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 5]
```diff
@@ -3982,12 +4020,37 @@ data:
     # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
     # new CRDs.
     #legacyCRDMirroring: true
+
+    # Enable usage reporting (telemetry) to VMware.
+    #enableUsageReporting: false
+    nodeIPAM:
```
Comment (knabben) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:4026]:
```suggestion
    #nodeIPAM:
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 6]
```diff
@@ -4271,7 +4335,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
-        image: antrea/antrea-ubuntu:v1.2.3
+        image: antrea/antrea-debian:v1.2.3_vmware.3
```
Comment (knabben) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:4338]:
```suggestion
        image: antrea/antrea-ubuntu:v1.2.3
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 7]
```diff
@@ -4331,7 +4396,8 @@ spec:
         - --log_file_max_num=4
         command:
         - start_ovs
-        image: antrea/antrea-ubuntu:v1.2.3
+        image: antrea/antrea-debian:v1.2.3_vmware.3
```
Comment (knabben) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:4399]:
```suggestion
        image: antrea/antrea-ubuntu:v1.2.3
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 8]
```diff
@@ -4365,7 +4431,8 @@ spec:
       initContainers:
       - command:
         - install_cni
-        image: antrea/antrea-ubuntu:v1.2.3
+        image: antrea/antrea-debian:v1.2.3_vmware.3
```
Comment (knabben) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:4434]:
```suggestion
        image: antrea/antrea-ubuntu:v1.2.3
```

File: addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml [hunk 9]
```diff
@@ -4388,6 +4455,29 @@ spec:
...
+        - --config
+        - /etc/antrea/antrea-agent-tweaker.conf
+        command:
+        - antrea-agent-tweaker
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        image: antrea/antrea-debian:v1.2.3_vmware.3
```
Comment (knabben) [inline addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml:4468]:
```suggestion
        image: antrea/antrea-ubuntu:v1.2.3
```

**Comment by knabben** [comment]
https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/antrea/1.2.3/package.yaml#L16
This package needs to be bumped and updated here.

**Comment by jayunit100** [comment]
closing this one, @knabben will carry this on

**Pull request closed without merging**